resource/closecaption_english.txt

```diff
@@ -3,4765 +3,4765 @@
+"wolfe.HoldingPosition0"	"<clr:154,205,50><B>Wolfe:<B> Covering area."
 
 
-"wolfe.HoldingPosition1"	"<clr:154,205,50>Wolfe: Holding position."
+"wolfe.HoldingPosition1"	"<clr:154,205,50><B>Wolfe:<B> Holding position."
 
 
-"wolfe.HoldingPosition2"	"<clr:154,205,50>Wolfe: I'm in place."
+"wolfe.HoldingPosition2"	"<clr:154,205,50><B>Wolfe:<B> I'm in place."
 
 
-"wolfe.HoldingPosition3"	"<clr:154,205,50>Wolfe: In position."
+"wolfe.HoldingPosition3"	"<clr:154,205,50><B>Wolfe:<B> In position."
 
 
-"wolfe.HoldingPosition4"	"<clr:154,205,50>Wolfe: Nothing's getting past me."
+"wolfe.HoldingPosition4"	"<clr:154,205,50><B>Wolfe:<B> Nothing's getting past me."
 
 
-"wolfe.HoldingPosition5"	"<clr:154,205,50>Wolfe: Secure area."
+"wolfe.HoldingPosition5"	"<clr:154,205,50><B>Wolfe:<B> Secure area."
 
 
-"wolfe.HoldingNorth0"	"<clr:154,205,50>Wolfe: Covering north."
+"wolfe.HoldingNorth0"	"<clr:154,205,50><B>Wolfe:<B> Covering north."
 
 
-"wolfe.HoldingNorth1"	"<clr:154,205,50>Wolfe: Covering north side."
+"wolfe.HoldingNorth1"	"<clr:154,205,50><B>Wolfe:<B> Covering north side."
 
 
-"wolfe.HoldingNorth2"	"<clr:154,205,50>Wolfe: I got north side."
+"wolfe.HoldingNorth2"	"<clr:154,205,50><B>Wolfe:<B> I got north side."
 
 
-"wolfe.HoldingNorth3"	"<clr:154,205,50>Wolfe: North side in place."
+"wolfe.HoldingNorth3"	"<clr:154,205,50><B>Wolfe:<B> North side in place."
 
 
-"wolfe.HoldingNorth4"	"<clr:154,205,50>Wolfe: North side is mine."
+"wolfe.HoldingNorth4"	"<clr:154,205,50><B>Wolfe:<B> North side is mine."
 
 
-"wolfe.HoldingNorth5"	"<clr:154,205,50>Wolfe: North side standing to."
+"wolfe.HoldingNorth5"	"<clr:154,205,50><B>Wolfe:<B> North side standing to."
 
 
-"wolfe.HoldingSouth0"	"<clr:154,205,50>Wolfe: Covering south."
+"wolfe.HoldingSouth0"	"<clr:154,205,50><B>Wolfe:<B> Covering south."
 
 
-"wolfe.HoldingSouth1"	"<clr:154,205,50>Wolfe: Covering south side."
+"wolfe.HoldingSouth1"	"<clr:154,205,50><B>Wolfe:<B> Covering south side."
 
 
-"wolfe.HoldingSouth2"	"<clr:154,205,50>Wolfe: I got south side."
+"wolfe.HoldingSouth2"	"<clr:154,205,50><B>Wolfe:<B> I got south side."
 
 
-"wolfe.HoldingSouth3"	"<clr:154,205,50>Wolfe: South side in place."
+"wolfe.HoldingSouth3"	"<clr:154,205,50><B>Wolfe:<B> South side in place."
 
 
-"wolfe.HoldingSouth4"	"<clr:154,205,50>Wolfe: South side is mine."
+"wolfe.HoldingSouth4"	"<clr:154,205,50><B>Wolfe:<B> South side is mine."
 
 
-"wolfe.HoldingSouth5"	"<clr:154,205,50>Wolfe: South side standing to."
+"wolfe.HoldingSouth5"	"<clr:154,205,50><B>Wolfe:<B> South side standing to."
 
 
-"wolfe.HoldingEast0"	"<clr:154,205,50>Wolfe: Covering east."
+"wolfe.HoldingEast0"	"<clr:154,205,50><B>Wolfe:<B> Covering east."
 
 
-"wolfe.HoldingEast1"	"<clr:154,205,50>Wolfe: Covering east side."
+"wolfe.HoldingEast1"	"<clr:154,205,50><B>Wolfe:<B> Covering east side."
 
 
-"wolfe.HoldingEast2"	"<clr:154,205,50>Wolfe: East side in place."
+"wolfe.HoldingEast2"	"<clr:154,205,50><B>Wolfe:<B> East side in place."
 
 
-"wolfe.HoldingEast3
```

Comment:
I think we should italicize this and add a SFX tag as it's a beep rather than the computer voice saying anything

Reply:
whoops, this ended up on the wrong line somehow. it should be on `BASE COMPUTER: [Time-Out]`

Reply:
sfx-tag, yes.
But why italic? I mean why are the sfx lines even in italic? They have brackets. I think this is notation enough for them.
So I would propose to even remove the current i's from the sfx section in the whole. How about that?

Reply:
Sure, that would be fine as well.

Reply:
I'll fix that right away.